src/index.js

```diff
@@ -1,55 +1,61 @@
+/* eslint import/extensions:0 */
+/* eslint import/no-unresolved:0 */
 import React, { Component } from 'react'
 import PropTypes from 'prop-types'
 import {
   Modal,
   View,
-  ListView,
+  FlatList,
   TouchableOpacity,
   Text,
   TextInput,
   KeyboardAvoidingView,
-  Platform
+  Platform,
+  SafeAreaView
 } from 'react-native'
 
-
-import styles from './styles'
-
+import styles from "./styles"
 
 export default class ModalFilterPicker extends Component {
-  constructor (props, ctx) {
+  static keyExtractor(item, index) {
```

Comment:
Does referencing this static method via `this.keyExtractor` work?

Reply:
@hiddentao This should be resolved by using `this.constructor.keyExtrator` in the latest commit (b316486).